recipes/snmf/meta.yaml

```diff
@@ -0,0 +1,26 @@
+about:
+  home: https://github.com/avengerpb/snmf_conda
+  license: Artistic-2.0
+  summary: Estimating individual admixture coefficients based on sparse non-negative matrix factorization and population genetics
+
+build:
+  number: 0
+  skip: False
+
+requirements:
+  build:
+    - perl-bioperl
+    - perl-lib
+
+package:
+  name: snmf
+  version: 1.0.0
+
+source:
+   fn: 1.5.tar.gz
+   url: https://github.com/avengerpb/snmf_conda/archive/1.5.tar.gz
```

Comment:
Sorry I'm just seeing this now, but how does the repo referred to here differ from http://membres-timc.imag.fr/Olivier.Francois/snmf/software.htm? Did you create a fork of the original sNMF? If so the recipe name should be changed to differentiate it from the original.